src/main/webapp/scripts/sys/isoFipsCountryMapScript.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * CU Generic ISO/FIPS Country modification
+ */
+const ISO_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.isoCountryCode';
+const ISO_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.isoCountry.name.div';
+const FIPS_COUNTRY_CODE_FIELD = 'document.newMaintainableObject.fipsCountryCode';
+const FIPS_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.fipsCountry.name.div';
+const EMPTY_STRING = '';
+
+function onblur_lookupFipsCountry(fipsCountryCodeField) {
+    let primaryKeyValue = ensurePrimaryKeyIsUppercase(fipsCountryCodeField);
+    if (primaryKeyValue === null || primaryKeyValue === '') {
+        clearCountryName(FIPS_COUNTRY_NAME_FIELD);
+    } else {
+        setFieldToSpecifiedValue(fipsCountryCodeField, primaryKeyValue);
+        updateCountryName(primaryKeyValue, FIPS_COUNTRY_NAME_FIELD);
+    }
+}
+
+function clearCountryName(countryNameFieldName) {
+    dwr.util.setValue(countryNameFieldName, EMPTY_STRING)
+}
+
+function setFieldToSpecifiedValue(fieldName, fieldValue) {
+    dwr.util.setValue(fieldName, fieldValue)
+}
+
+function ensurePrimaryKeyIsUppercase(countryCodeField) {
+    if (countryCodeField === null || countryCodeField === '') {
+        return countryCodeField;
+    }
+    return dwr.util.getValue(countryCodeField).trim().toUpperCase();
+}
+
+function updateCountryName(primaryKeyValue, targetFieldName) {   
+    if (primaryKeyValue === '') {
+        clearCountryName(targetFieldName);
+    } else {
+        var dwrReply = {
```

Comment:
Can the various dwrReply variables be declared using "const" or "let" instead of "var"?

Reply:
Ok, thanks for explaining. I'm okay with leaving these particular variables as "var" ones for now.